runtime/doc/lsp.txt

```diff
@@ -1128,9 +1128,12 @@ code_action({options})                             *vim.lsp.buf.code_action()*
                      is applied without user query.
                    • range: (table|nil) Range for which code actions should be
                      requested. If in visual mode this defaults to the active
-                     selection. Table must contain `start` and `end` keys with
-                     {row, col} tuples using mark-like indexing. See
+                     selection. Table must contain `start` and `end` keys with {row, col} tuples using mark-like indexing. See
                      |api-indexing|
+                     • silent: (boolean|nil) When `true`, don't show a
+                       notification when no code actions are available
+                     • select: (function|nil) See |vim.ui.select| Custom
```

Comment:
vim.ui.select exists to be overridden, but here we're providing another way to override it? 

if the code_action impl sets `kind=code_action` (like https://github.com/neovim/neovim/pull/23889 ) would that avoid the need for this?